src/modules/core/components/InfoPopover/MemberInfoPopover/UserPermissions.tsx

```diff
@@ -0,0 +1,48 @@
+import React from 'react';
+import { ColonyRole } from '@colony/colony-js';
+import { defineMessages } from 'react-intl';
+
+import PermissionsLabel from '~core/PermissionsLabel';
+
+import styles from './MemberInfoPopover.css';
+import Heading from '~core/Heading';
+
+const displayName = `InfoPopover.MemberInfoPopover.UserPermissions`;
+
+interface Props {
+  roles: ColonyRole[];
+}
+
+const MSG = defineMessages({
+  labelText: {
+    id: 'InfoPopover.MemberInfoPopover.UserPermissions.labelText',
+    defaultMessage: 'Permissions',
+  },
+});
+
+const UserPermissions = ({ roles }: Props) => {
+  return (
+    <div className={styles.permissionsContainer}>
+      <Heading
+        appearance={{
+          size: 'normal',
```

Comment:
Also, it looks to me like the font size is smaller in this screenshot:

![Screenshot from 2021-09-02 22-13-31](https://user-images.githubusercontent.com/1193222/131902889-97af9d2b-37ab-4a67-8fb9-2e53b5f78826.png)

Please compare with the Figma file and adjust it accordingly